test/testProjects/containerapps-albumapi-javascript/src/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:lts-alpine

```

Comment:
Have you verified that making this Dockerfile invalid breaks the tests?

Reply:
Thinking about this more, it might be good to explore making a test case with a broken Dockerfile on purpose so we have a consistent and tested experience for that.

Reply:
Great suggestion, I'm working on an update to address this here!
https://github.com/microsoft/vscode-azurecontainerapps/pull/708